### ISSUE TITLE
Can someone provide a python script that can allow me to train the model from scratch?

### ISSUE THREAD

**Issue by smallcube** [lucidrains/stylegan2-pytorch#217]
In the stylegan2_pytorch.py, I just can't find the place to set some hyper-parameters, for example, num_training_step.

**Comment by lucidrains** [comment]
@smallcube It is a one-line command `stylegan2_pytorch --data /path/to/your/images`

**Comment by smallcube** [comment]
> @smallcube It is a one-line command `stylegan2_pytorch --data /path/to/your/images`

I just want to make some customed settings, e.g., new loss function, so a script that allows me to run the code from scratch can be helpful. By the way, excellent work.

**Comment by DWendou** [comment]
i have same question，how to run stylegan2_pytorch.py to train on my datasets，i also can't find the place to set some hyper-para，i also want to get FID loss curve，but i can't do it
